crates/rolldown/tests/fixtures/function/experimental/strict_execution_order/esbuild_issue_399/strict/init-dep-2.js

```diff
@@ -0,0 +1,3 @@
+global.foo = {
+  log: () => console.log("foo.log() (from entry 2) called"),
```

Comment:
Can you mock the function and do some assertion? since `console.log` is useless in unit testing, https://sevic.dev/notes/spies-mocking-node-test-runner/, I just found node has builtin support.

Reply:
I was going to, but I don't want change the original input that copied from issues. These tests are just examples to show output. We'll add more serious tests once the feature is going be landed stablely.

Reply:
yeah, just wanna this feature to be stable, you can go ahead and improve in the following pr.